src/dstack/_internal/core/models/instances.py

```diff
@@ -110,6 +110,11 @@ def get_public_keys(self) -> List[str]:
         return [ssh_key.public.strip() for ssh_key in self.ssh_keys]
 
 
+class InstanceSharedInfo(CoreModel):
+    total_blocks: Union[Literal["auto"], int]
+    busy_blocks: int = 0
```

Comment:
Not sure about storing `busy_blocks` in json vs in a separate column. Any particular reasons to do so? I'm generally inclined to keep static data in json and changing fields like this as separate columns.

Maybe using json is more flexible if we take a complete different approach instead of relying on `blocks`, Idk.

Reply:
Replaced with two integer columns https://github.com/dstackai/dstack/pull/2253/commits/8b555f7a6ad65afbed244e3d1e76359bdbbcbbf9